cfme/tests/intelligence/reports/test_reports_manual.py

```diff
@@ -260,3 +260,129 @@ def test_reports_online_vms():
         1504010
     """
     pass
+
+
+@test_requirements.report
+@pytest.mark.manual
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1743579])
+def test_created_on_time_report_field():
+    """
+    Bugzilla:
+        1743579
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: medium
+        initialEstimate: 1/2h
+        setup:
+            1. Add a provider and provision a VM
+        testSteps:
+            1. Create a report with `Created on Time` field.
```

Comment:
Could you add little more info about what report/what page to navigate to? Maybe name of sample report that usually includes `Created on Time` field?